src/cmd_line/commands/tab.ts

```diff
@@ -124,20 +125,22 @@ export class TabCommand extends node.CommandBase {
       case Tab.Only:
         await vscode.commands.executeCommand('workbench.action.closeOtherEditors');
         break;
-      case Tab.Move:
-        if (this.arguments.count === 0) {
-          await vscode.commands.executeCommand('moveActiveEditor', { to: 'first' });
-        } else if (this.arguments.count === undefined) {
-          await vscode.commands.executeCommand('moveActiveEditor', { to: 'last' });
-        } else {
-          await vscode.commands.executeCommand('moveActiveEditor', {
-            to: 'position',
-            by: 'tab',
-            value: this.arguments.count + 1,
-          });
-        }
+      case Tab.Move: {
+        const args = (() => {
```

Comment:
It's not too common, though I personally use it a lot in my code, primarily to avoid mutable variables (especially in larger code blocks). I don't mind changing this to a more familiar syntax if requested.

Reply:
Yeah, frankly, I like what you're trying to do (single assignment is much easier to reason about in a lot of cases), but given that it's not a pattern in this code base, I think just using `let` would be better here.